tests/unit/lms/views/exceptions_test.py

```diff
@@ -99,11 +83,6 @@ class TestError(ExceptionViewTest):
     exception = None
 
     response_status = 500
-    # Although I don't think it would do any harm (sentry_sdk seems smart
-    # enough to not double report the exception to Sentry) we don't need to
-    # call report_exception() in the case of a non-HTTPError exception
-    # because Sentry's Pyramid integration does it for us automatically.
```

Comment:
This comment was incorrect. It's the `500` status that causes the exception to be automatically reported, not the exception class.